packages/mediapipe-core/lib/src/io/task_result.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+import 'package:mediapipe_core/interface.dart';
+import 'package:mediapipe_core/io.dart';
+import 'third_party/mediapipe/generated/mediapipe_common_bindings.dart'
+    as bindings;
+
+/// Anchor class for native memory implementations of task results.
+mixin TaskResult {}
```

Comment:
What is this for? I am not sure. 🙃

Reply:
The following code is in `task_executor.dart`:

```dart
abstract class TaskExecutor<..., Result extends TaskResult> {}
```

I could be talked out of this, but felt it was a little nicer to make the following impossible:

```dart
class TextClassificationExecutor extends TaskExecutor<..., WrongType> {}
```